test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlCustomQueryTests.cs

```diff
@@ -49,16 +49,13 @@ public async Task GivenASqlQuery_IfAStoredProcExistsWithMatchingHash_ThenStoredP
             // set the wait time to 1 second
             CustomQueries.WaitTime = 1;
 
-            // Create a query with a known hash
             var queryParameters = new[]
             {
-                Tuple.Create("participating-organization.identifier", $"TAB%7C910000290"),
-                Tuple.Create(SearchParameterNames.Include, $"OrganizationAffiliation:location"),
-                Tuple.Create("participating-organization.type", $"practice"),
```

Comment:
Who was this test changed?

Reply:
Unit test case was failing, needed modification to pass unit test.